test/e2e/support/utils/visit-admin-page.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Node dependencies
+ */
+import { join } from 'path';
+
+/**
+ * Internal dependencies
+ */
+import { createURL } from './create-url';
+import { isCurrentURL } from './is-current-url';
+import { loginUser } from './login-user';
+
+/**
+ * Visits admin page; if user is not logged in then it logging in it first, then visits admin page.
+ * @param {string} adminPath String to be serialized as pathname.
+ * @param {string} query String to be serialized as query portion of URL.
```

Comment:
It would be a nicer API if these weren't strings but rather a key/value object. It'd make it a lot easier to build/manipulate any query args in JS. So you could do:

```js
visitAdminPage( 'post-new.php', { post_type: 'wp_block' } );
```

instead of:

```js
visitAdminPage( 'post-new.php', 'post_type=wp_block' );
```

Might be worth tackling before this gets published as a package to keep the external API stable 😄

Reply:
Yeah, makes sense. I will open its own PR for that.